.circleci/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Upgrade the helm client version here when the az cluster tiller version
```

Comment:
this is a pretty clever dockerfile ;)

Reply:
Heh thx 😄 For future ref, Dockerfile moved to https://github.com/helm/docker-kubectl-helm-az 🔍🕵️‍♂️